deploy/olm-catalog/namespace-configuration-operator/0.0.1/namespace-configuration-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"redhatcop.redhat.io/v1alpha1","kind":"NamespaceConfig","metadata":{"name":"example-namespaceconfig"},"spec":{"size":3}}]'
+    capabilities: Basic Install
+  name: namespace-configuration-operator.v0.0.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: NamespaceConfig
+      name: namespaceconfigs.redhatcop.redhat.io
+      version: v1alpha1
+  description: Placeholder description
```

Comment:
Did you intend to keep the description as a placeholder?

Reply:
this is not the file I committed... I don't understand.

Reply:
that file should not have been there, thanks for catching it.